config.py

```diff
@@ -7,14 +7,14 @@ class Config(object):
     PORT = os.getenv('PORT', 8085)
     LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
     USE_SESSION_FOR_NEXT = True
-
-    BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL', 'http://localhost:8001/backstage-api')
     RESPONSE_OPERATIONS_UI_SECRET = os.getenv('RESPONSE_OPERATIONS_UI_SECRET', "secret")
+    BACKSTAGE_BASE_URL = os.getenv('BACKSTAGE_BASE_URL')
```

Comment:
We should be cautious when changing the name of environmental variables. We will either need to change this back or propagate these changes to all of the jenkins deployments jobs; dev,ci,int,sit,pre-prod,prod and the docker repo: https://github.com/ONSdigital/ras-rm-docker-dev